### PR TITLE
blender: update to 2.83

### PR DESCRIPTION
**Pull request by teldra**
also fix openimagedenoise is just for glibc x86_64

### CODE REVIEW

File: srcpkgs/blender/template [hunk 1]
```diff
@@ -54,8 +54,8 @@ case "$XBPS_TARGET_MACHINE" in
 esac
 
 case "$XBPS_TARGET_MACHINE" in
-x86_64*)
-	# Enable Intel Open Image Denoise on x86_64
+x86_64)
+	# Intel(R) Open Image Denoise supports glibc 64-bit platforms only
```
Comment (Piraty) [inline srcpkgs/blender/template:58]: yet srcpkgs/openimagedenoise-devel/template has `archs="x86_64*"` set (not pinning it to glibc), can you clarify and add a fix to either one of them?

Comment (teldra) [reply srcpkgs/blender/template:58]: https://github.com/void-linux/void-packages/pull/22562 here i did this.

**Comment by teldra** [comment]
I don't have the time in the near future to debug this package.

**Comment by mvf** [comment]
Continued in https://github.com/void-linux/void-packages/pull/22792

**Pull request closed without merging**